packages/web/src/components/collapsible-content/CollapsibleContent.tsx

```diff
@@ -52,17 +54,21 @@ export const CollapsibleContent = ({
       >
         <div ref={ref}>{children}</div>
       </div>
-      <Button
+      <PlainButton
         className={cn(styles.toggleCollapsedButton, toggleButtonClassName)}
-        iconClassName={styles.toggleCollapsedButtonIcon}
+        css={{
+          paddingTop: spacing.l,
+          paddingBottom: spacing.l,
+          margin: '0 auto'
```

Comment:
"fullWidth" ?

Reply:
sadly this component is weird, it'll clip off half the button unless i apply these styles. tried fullwidth instead and it did the same